sale_amendment/view/sale_order.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+
+    <record id="view_order_form" model="ir.ui.view">
+      <field name="name">sale.order.form.amendment</field>
+      <field name="model">sale.order</field>
+      <field name="inherit_id" ref="sale_stock.view_order_form_inherit"/>
+      <field name="arch" type="xml">
+         <button name="ship_recreate">
```

Comment:
relying on the default value of position... :frowning: explicit is better that explicit IMO (and I never remember that default value...)